ledger/src/shred/merkle_tree.rs

```diff
@@ -9,6 +9,8 @@ pub(crate) const SIZE_OF_MERKLE_ROOT: usize = std::mem::size_of::<Hash>();
 const_assert_eq!(SIZE_OF_MERKLE_ROOT, 32);
 const_assert_eq!(SIZE_OF_MERKLE_PROOF_ENTRY, 20);
 pub(crate) const SIZE_OF_MERKLE_PROOF_ENTRY: usize = std::mem::size_of::<MerkleProofEntry>();
+// Number of proof entries for the standard 64 shred batch.
+pub(crate) const PROOF_NUM_ENTRIES_TYPICAL: u8 = 6;
```

Comment:
Should we be more explicit ?
```suggestion
pub(crate) const PROOF_ENTRIES_FOR_32_32_BATCH: u8 = 6;
```

Reply:
yeah, I considered this and for some reason decided against it. You're convincing being explicit is better